src/layers/models/layersManager.ts

```diff
@@ -278,6 +280,27 @@ export class LayersManager {
     this.requestCreateLayerCounter?.inc({ requestType: 'CreateLayer', jobType });
   }
 
+  //TODO: decide what the function will return -void or boolean
```

Comment:
why?
you can call this method within an outside try-catch  
passes return 200 with isValid = true, reason = undefined
if 400 return 200 with isValid = false and reasoun = error message
else (500) throw the exception to the client...